### ISSUE TITLE
[Bug] "Go back" in browser does not initialize the form correctly

### ISSUE THREAD

**Issue by Polfo** [Laravel-Backpack/devtools-issues#22]
# Bug report

### What I did

http://127.0.0.1:8000/admin/devtools/model/create
add one column, probably other things also
uncheck Run Migration
uncheck Generate Crud

### What I expected to happen

no error

### What happened

```
ErrorException
include(D:\laragon\www\dauwgard.be\vendor\composer/../../app/Models/Test.php): Failed to open stream: No such file or directory
http://127.0.0.1:8000/admin/devtools/model
```

It is not the first time I have this error, but I don't know yet how to reproduce it.

### What I've already tried to fix it

Navigate back, relaod page, try again with different model name, now I get
```
Symfony\Component\Yaml\Exception\ParseException
Unable to parse at line 4 (near ": string").
http://127.0.0.1:8000/admin/devtools/model
```

### Backpack, Laravel, PHP, DB version

When I run ```php artisan backpack:version``` the output is:
### PHP VERSION:
PHP 8.0.2 (cli) (built: Feb  3 2021 18:36:40) ( ZTS Visual C++ 2019 x64 )
Copyright (c) The PHP Group
Zend Engine v4.0.2, Copyright (c) Zend Technologies
    with Zend OPcache v8.0.2, Copyright (c), by Zend Technologies

### LARAVEL VERSION:
v8.74.0@004ea195012d5132eca07a176a6e587c6a74815e

### BACKPACK VERSION:
4.1.61@4400187391eb5b73672c2b166a990cd737277409

### Operating System and Server Setup

I encountered the problem above on my [Windows 10] machine, and my web server is set up using [php artisan serve].

**Comment by tabacitu** [comment]
@promatik again, your debugging / copy-pasting the form input would have been super-useful here...

**Comment by pxpm** [comment]
We need that for sure. @tabacitu 

I am unable to reproduce this. I tried with `Laragon` and also with `php artisan serve`. Win 10 and PHP 8.1. 

Well, technically there are two different issues, first, when creating, we have checks in place that guarantees us that if no files are created or there are errors creating the files we abort with 500 and a nice message. So this one I could not reproduce.

BUT after I created the model, it went to show me the files, if I hit the "go back" in browser the interface will have messed values that will break our code. For example the timestamps column will have the `string` type. I need further investigation on this one, just letting you guys know what I've found.
